src/components/ImageWithSizeCalculation.js

```diff
@@ -112,15 +92,21 @@ class ImageWithSizeCalculation extends PureComponent {
                     this.props.style,
                 ]}
             >
-                <Image
+                <FastImage
                     style={[
                         styles.w100,
                         styles.h100,
                     ]}
-                    source={{uri: this.props.url}}
-                    resizeMode="contain"
+                    fallback
```

Comment:
I thought we only potentially wanted to use this prop on web? Since doesn't this make us use react-native's `Image` component instead of the library?

Reply:
Oh good spot - left that there from my initial tests with the `fallback` prop, will remove and verify it works correctly 🙌